app/models/event.rb

```diff
@@ -159,10 +160,15 @@ def as_indexed_json(options={})
       "updated_at" => updated_at,
       "indexed_at" => indexed_at,
       "occurred_at" => occurred_at,
+      "citation_id" => citation_id,
       "cache_key" => cache_key
     }
   end
 
+  def citation_id
+    [subj_id, obj_id].compact.sort.join("-")
```

Comment:
@kjgarza why do you use `compact`? Both subj_id and obj_id are required.

Reply:
corrrect, addressed